### PR TITLE
Search token by symbol and address for token importing from URL

### LINKED ISSUE #1766: No import token modal when type a token into URL
Paste the link into a browser's tab: https://swap.cow.fi/#/1/swap/WETH/DEC

**AR**: To token is emprty, there is no request to import DEC token

**ER**: import token modal should appear

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Fixes #1766

In case, when the URL contains a token symbol of token from the inactive list we should automatically import the token.
In this fix, the algorithm of the searching token was changed. Before the fix the search was considering only token addresses, but not symbols.

image

  # To Test

See #1766

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
Great job!

**Review by alfetopito — APPROVED** [review]

File: src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx [hunk 1]
```diff
@@ -16,11 +16,16 @@ export function ImportTokenModal(props: ImportTokenModalProps) {
...
   const tradeState = useTradeState()
-  // token warning stuff
-  const [loadedInputCurrency, loadedOutputCurrency] = [
-    useCurrency(tradeState?.state?.inputCurrencyId),
-    useCurrency(tradeState?.state?.outputCurrencyId),
-  ]
+  const loadedInputCurrency = useSearchInactiveTokenLists(
+    tradeState?.state?.inputCurrencyId || undefined,
+    1,
+    true //
```
Comment (anxolin) [inline src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx:22]: U were going to write a comment here?

**Review by anxolin — COMMENTED** [review]
One question, what happens if there's 2 tokens matching the symbol but has different address? 

`awbtc` is twice, but I think it has the same address, but it could be that it is not the case. Which one is selected?

File: src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx (hunk 1 above)
Comment (shoom3301) [reply src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx:22]: No, it's to keep the line break. Without the comment prettier make it in one line (and it looks not so good)

**Comment by shoom3301** [comment]
> One question, what happens if there's 2 tokens matching the symbol but has different address?
> 
> `awbtc` is twice, but I think it has the same address, but it could be that it is not the case. Which one is selected?

It's very good question :) 
The first found will be selected. Anyway, that case is pretty artificial. If you select `awbtc` from UI (not URL) the token address will be used instead of the symbol.

**There is a rule: when we have two or more tokens with the same symbol - we use their addresses instead of symbols.**
`useAreThereTokensWithSameSymbol()`

**Review by anxolin — APPROVED** [review]

**Pull request merged**